templates/taxi/car_form.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update ,Create " }}Car</h1>
+
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input class="btn btn-primary" type="submit" value="Submit">

```

Comment:
Consider adding a `Cancel` button to improve user experience. This button should redirect the user to the previous page or a relevant list page.